collector.go

```diff
@@ -67,12 +122,33 @@ type ChunkedCollector struct {
 	started, stopped bool
 	stopChan         chan struct{}
 
+	queueSizeBytes  uint64
 	pendingBySpanID map[SpanID]Annotations
 
 	// mu protects pendingBySpanID, lastErr, started, stopped, and stopChan.
 	mu sync.Mutex
 }
 
+// NewChunkedCollector is shorthand for:
+//
+// 	c := &ChunkedCollector{
+// 		Collector:    c,
+// 		MinInterval:  500 * time.Millisecond,
+// 		FlushTimeout: 50 * time.Millisecond,
+// 		MaxQueueSize: 32 * 1024 * 1024, // 32MB
+// 		Log:          log.New(os.Stderr, "appdash: ", log.LstdFlags),
+// 	}
```

Comment:
We've discussed this, and unfortunately no better way to have the docs communicate that this `NewChunkedCollector` does not initialize private fields (therefore, it's completely optional to use, and it's perfectly fine to use `ChunkedCollector` directly) other than to keep it as is.

Hopefully we'll avoid the cat situation like above because the code and docs are right next to each other, and should be easier to keep in sync.